src/pybind/mgr/dashboard/model/feedback.py

```diff
@@ -0,0 +1,33 @@
+# # -*- coding: utf-8 -*-
+
+class FeedbackModel:
+    project_id: int
+    tracker_id: int
+    subject: str
+    description: str
+    severity: str
+    status: int
+
+    def __init__(self, project_id, tracker_id, subject, description, severity):
+        self.project_id = int(project_id)
+        self.tracker_id = int(tracker_id)
+        self.subject = subject
+        self.description = description
+        self.severity = severity
+        self.status = 1
```

Comment:
I'd replace this with an Enum. A numeric literal is not really nice for reviewers or future contributors. If you replace this with `Status.NEW` that's easier to follow and extend.